R/finalize_plot.R

```diff
@@ -117,6 +121,7 @@ finalize_plot <- function(plot = NULL,
                           width = 670/72, # comms spec: 670px @ 72ppi
                           height = 400/72, # comms spec: 400px @ 72ppi
                           title_width = NULL, # if unspecified, default to width/4
+                          no_title = FALSE,
```

Comment:
The motivation was to address both of those cases. I think your proposal for both seems like it would work, and I agree that a general case like `inherit = ""` would make sense in the long term. It also doesn't require very complex code additions. I can take a look later this week.

Reply:
I've added this functionality.